src/filters/index.js

```diff
@@ -61,7 +61,7 @@ var digitsRE = /(\d{3})(?=\d)/g
 exports.currency = function (value, currency) {
   value = parseFloat(value)
   if (!isFinite(value) || (!value && value !== 0)) return ''
-  currency = currency || '$'
+  currency = currency === false ? '' : currency || '$'
```

Comment:
so that he can do `money | currency ''`. Currently empty string is considered falsy and gets casted into `$`.